ui/packages/consul-ui/app/components/consul/acl/disabled/index.hbs

```diff
@@ -0,0 +1,28 @@
+<AppView>
+  <BlockSlot @name="header">
+    <h1>
+      Tokens
+    </h1>
+  </BlockSlot>
+  <BlockSlot @name="content">
+    <EmptyState data-test-acls-disabled>
+      <BlockSlot @name="header">
+        <h2>Welcome to ACLs</h2>
+      </BlockSlot>
+      <BlockSlot @name="body">
+        <p>
+          ACLs are not enabled in this Consul cluster. We strongly encourage the use of ACLs in production environments for the best security practices.
+        </p>
+      </BlockSlot>
+      <BlockSlot @name="actions">
+        <li class="docs-link">
+          <a href="{{env 'CONSUL_DOCS_URL'}}/acl/index.html" rel="noopener noreferrer" target="_blank">Read the documentation</a>
+        </li>
+        <li class="learn-link">
+          <a href="{{env 'CONSUL_DOCS_LEARN_URL'}}/consul/security-networking/production-acls" rel="noopener noreferrer" target="_blank">Follow the guide</a>
+        </li>
+      </BlockSlot>
+    </EmptyState>
+  </BlockSlot>
+</AppView>
+
```

Comment:
We now have an 'ACLs disabled' component instead of building it into AppView. I've left the old on in AppView for the moment. It will probably be removed as part of the removal of legacy ACLs.